base_layer/core/src/blocks/block.rs

```diff
@@ -74,7 +74,7 @@ impl Block {
     }
 
     // create a total_coinbase offset containing all fees for the validation
-    fn calculate_coinbase_and_fees(&self, block_reward: MicroTari) -> MicroTari {
+    pub fn calculate_coinbase_and_fees(&self, block_reward: MicroTari) -> MicroTari {
```

Comment:
Instead of just making this pvt fn public, I think it's better to:

- rename to `pub calculate_total_coinbase_value`
- bring the emission schedule calc into here from `check_internal_consistency`